bindings/python/scripts/dlite-validate

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+"""Validator for dlite datamodels.
+"""
+import argparse
+import json
+import re
+import yaml
+from pathlib import Path
+
+import dlite
+
+
+def parse(url, driver, options=None):
+    """Parse url and return an instance."""
+    match = re.match(r'^([a-zA-Z][a-zA-Z0-9+.-]*)://', url)
+    if match and match.groups[0].lower() != 'file':
```

Comment:
There seems to be an error here: match.groups()
```suggestion
    if match and match.groups()[0].lower() != 'file':
```

Reply:
Well spottet!